apps/backend/src/main/resources/api.yaml

```diff
@@ -13,4 +13,108 @@ paths:
     get:
       responses:
         400:
-          description: Bad request
\ No newline at end of file
+          description: Bad request
+components:
+  schemas:
+    ModelDTO:
+      required:
+        - description
+        - name
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    LocationDTO:
+      required:
+        - description
+        - name
+        - parentLocationId
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+        parentLocationId:
+          type: integer
+          nullable: true
+    ItemDTO:
+      required:
+        - modelId
+        - movementsId
+      properties:
+        id:
+          type: integer
+        modelId:
+          type: integer
+        movementsId:
+          type: array
+          items:
+            type: integer
+    MovementDTO:
+      required:
+        - locationId
+        - itemId
+        - movementType
+      properties:
+        id:
+          type: integer
+        locationId:
+          type: integer
+        itemId:
+          type: integer
+        movementType:
+          type: string
```

Comment:
A date would be be nice init !

Reply:
> On a general note I like havin created and updated date on all things, but not sure it would be usefull in the API.
No

Reply:
I agree with @nigelmann.
`id`, `createdDate` and `updatedDate` on all managed entities (not necessary these names).
They are not needed on _pure_ association tables.